api/types/accesslist/inclusion.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package accesslist
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gravitational/trace"
+)
+
+// Inclusion values indicate how membership and ownership of an AccessList
+// should be applied.
+type Inclusion uint
```

Comment:
Do we still need the string->int conversions? We can use the string as-is now, right?

Reply:
Yeah; we have two definitions of the AccessList types - the protobuf-generated wire format that uses strings, and the internal version that uses the int types. This `uint` based Inclusion type is for the internal side.

Reply:
I don't follow it - why do we have to transform the string to an int and then back again? Since we decided to live with the proto strings, why not `type Inclusion string`, if a custom type is necessary at all?

It looks like we could simplify this greatly, ditch all the custom marshalers and all of inclusion_test.go as well.

Reply:
Removed entirely